internal_functions/work_with_fastq.py

```diff
@@ -0,0 +1,101 @@
+def gc_count_filter(sequences: list[str], gc_bounds: int) -> list[bool]:
+    """
+    Filter sequences by parameter gc_bounds.
+
+    Arguments:
+    - sequences (list[str]): list of sequences
+    - gc_bounds (int): the GC interval of the composition (in percent) for filtering (by default is (0, 100)
+
+
+    Return:
+    - a list of boolean values, where true corresponds to the passage of the filtering element
+      by parameter gc_bounds
+    """
+    gc = []
+    gc_answer = []
+    for sequence in range(len(sequences)):
+        gc.append((sequences[sequence].upper().count('G') + sequences[sequence].upper().count('C')) / len(
+            sequences[sequence]) * 100)
+
+    for gc_content in gc:
+
+        if type(gc_bounds) == int:
```

Comment:
Лучше использовать встроенную функцию в питоне для проверки типов isinstance():

```suggestion
        if isinstance(gc_bounds, int)
```

О, прикольная штука: с Python 3.10 в функцию isinstance можно передать несколько типов сразу:

```suggestion
        if isinstance(gc_bounds, int | float)
```
А вот тут проблема есть еще в том, что ты не учла, что пользователь мог подать float (см. условие задания), сейчас код падает с ошибкой.